policyengine_uk/variables/gov/dfe/extended_childcare_entitlement/extended_childcare_entitlement.py

```diff
@@ -27,6 +27,16 @@ def formula(benunit, period, parameters):
         # Use the appropriate hours based on the condition
         weekly_hours_to_use = min_(max_hours_used, weekly_hours_per_child)
 
+        # Get the hours usage fraction for this benefit unit
+        hours_usage_fraction = benunit(
+            "extended_childcare_hours_usage_fraction", period
+        )
+
+        # Apply the fraction to the weekly hours
+        weekly_hours_to_use = weekly_hours_to_use * benunit.project(
```

Comment:
If we are changing this to a 'maximum' usage rather than 'percentage of hours' then we'll need to instead:

```suggestion
        weekly_hours_to_use = min_(weekly_hours_to_use, benunit.project(maximum_extended_childcare_usage_fraction))
```

Reply:
done.